--- geany-1.23/src/keyfile.c.orig	2013-06-14 14:52:48.442473613 +0400
+++ geany-1.23/src/keyfile.c	2013-06-14 14:53:28.866469739 +0400
@@ -71,15 +71,15 @@
 #ifdef G_OS_WIN32
 #define GEANY_DEFAULT_TOOLS_TERMINAL	"cmd.exe /Q /C %c"
 #else
-#define GEANY_DEFAULT_TOOLS_TERMINAL	"xterm -e \"/bin/sh %c\""
+#define GEANY_DEFAULT_TOOLS_TERMINAL	"lxterminal"
 #endif
 #define GEANY_DEFAULT_TOOLS_BROWSER		"firefox"
 #define GEANY_DEFAULT_TOOLS_PRINTCMD	"lpr"
 #define GEANY_DEFAULT_TOOLS_GREP		"grep"
 #define GEANY_DEFAULT_MRU_LENGTH		10
-#define GEANY_DEFAULT_FONT_SYMBOL_LIST	"Sans 9"
-#define GEANY_DEFAULT_FONT_MSG_WINDOW	"Sans 9"
-#define GEANY_DEFAULT_FONT_EDITOR		"Monospace 10"
+#define GEANY_DEFAULT_FONT_SYMBOL_LIST	"Droid Sans 9"
+#define GEANY_DEFAULT_FONT_MSG_WINDOW	"Droid Sans 9"
+#define GEANY_DEFAULT_FONT_EDITOR		"Droid Sans Mono 10"
 #define GEANY_TOGGLE_MARK				"~ "
 #define GEANY_MAX_AUTOCOMPLETE_WORDS	30
 #define GEANY_MAX_SYMBOLS_UPDATE_FREQ	250
